packages/definition-runtime/README.md

```diff
@@ -1,5 +1,40 @@
 # @scale-codec/definition-runtime ![build status](https://img.shields.io/github/checks-status/soramitsu/scale-codec-js-library/master) ![version](https://img.shields.io/npm/v/@scale-codec/definition-runtime) ![license](https://img.shields.io/npm/l/@scale-codec/definition-runtime)
 
-Runtime for `@scale-codec/definition-compiler`.
+Runtime for `@scale-codec/definition-compiler`. Includes standard codecs (such as `U8`, `Str`, `Bool`, `Unit` etc), codec factories and [logging facilities](#tracking-and-logging).
 
-[Documentation](https://soramitsu.github.io/scale-codec-js-library/guide/compiler)
+## Installation
+
+Available on NPM:
+
+```shell
+npm i @scale-codec/definition-compiler
+```
+
+## Usage
+
+Usually, it is paired with the output of the compiler - it is the library that the compiled code imports utilities from.
+
+For advanced usage you can explore the package's [API](https://soramitsu.github.io/scale-codec-js-library/api/modules/scale_codec_definition_runtime).
+
+## Tracking and Logging
+
+All codecs use tracking API. This API could be used to track how decoding (there is no need to track encoding for now) goes, which might be useful for debugging. In order to see what is happening, a _tracker_ should be mounted. No any tracker is mounted by default, and the runtime cost in that case is minimal.
```

Comment:
```suggestion
All codecs use tracking API. For now, there is no need to track encoding, but you can use this API to track the decoding process. This might be useful for debugging purposes. To see what is happening, a _tracker_ should be mounted. Tracker are not mounted by default, and the runtime cost in that case is minimal.
```

Reply:
> <...> **Tracker are not** mounted by defaeult, <...>

Why "are" and not "is"?

Reply:
I meant to write tracker**s**. Edited the suggestion above